.github/workflows/lint.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     env:
-      GOVER: '1.22.2'
```

Comment:
Is there a specific formatter you're using for this?

IMO if we're going to spend time on minor formatting changes like this, it should be automated.

Reply:
I will have a separate PR later (hopefully in this sprint) and probably a pipeline step to do the formatting for us.

I use prettier on VS Code by the way.